bundles/automation/org.eclipse.smarthome.automation.importers/src/main/java/org/eclipse/smarthome/automation/internal/importers/Activator.java

```diff
@@ -0,0 +1,164 @@
+/**
+ * Copyright (c) 1997, 2015 by ProSyst Software GmbH and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.automation.internal.importers;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.eclipse.smarthome.automation.parser.Parser;
+import org.eclipse.smarthome.automation.template.TemplateProvider;
+import org.eclipse.smarthome.automation.template.TemplateRegistry;
+import org.eclipse.smarthome.automation.type.ModuleTypeProvider;
+import org.eclipse.smarthome.automation.type.ModuleTypeRegistry;
+import org.osgi.framework.BundleActivator;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.Filter;
+import org.osgi.framework.InvalidSyntaxException;
+import org.osgi.framework.ServiceReference;
+import org.osgi.util.tracker.ServiceTracker;
+import org.osgi.util.tracker.ServiceTrackerCustomizer;
+
+/**
+ * This class is an activator of this bundle. Opens the all used service trackers and registers the services -
+ * {@link ModuleTypeProvider} and {@link TemplateProvider}.
+ *
+ * @author Ana Dimova - Initial Contribution
+ *
+ */
+@SuppressWarnings("rawtypes")
+public class Activator implements ServiceTrackerCustomizer, BundleActivator {
+
+    private static final String MODULE_TYPES_ROOT = "automation/moduletype";
+    private static final String TEMPLATES_ROOT = "automation/template";
+
+    private ModuleTypeImporter mti;
+    private TemplateImporter ti;
+    private ServiceTracker tracker;
+    private BundleContext bc;
+
+    /**
+     * This method initialize importers for importing automation objects from local file system.
+     */
+    @SuppressWarnings("unchecked")
+    @Override
+    public void start(BundleContext bc) throws Exception {
+        this.bc = bc;
+        try {
+            Filter filter = bc.createFilter("(|(objectClass=" + TemplateRegistry.class.getName() + ")(objectClass="
```

Comment:
This bundle must not depend on either the template nor the module type registry. It should merely register providers as a service.